packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+## 44.15.0 - January 10, 2019
+
+### Added
+
+#### salesforcedx-vscode
+
+- New [Code of Conduct](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/CODE_OF_CONDUCT.md) ([PR #846](https://github.com/forcedotcom/salesforcedx-vscode/pull/846))
+
+#### salesforcedx-vscode-apex
+
+- Apex Tests sidebar displays results for tests executed using code lens, Apex Tests sidebar or command palette commands ([PR #800](https://github.com/forcedotcom/salesforcedx-vscode/pull/800))
+
+- Update right-click menu label based on test status in Apex Tests sidebar ([PR #811](https://github.com/forcedotcom/salesforcedx-vscode/pull/811))
+
+#### salesforcedx-vscode-core
+
+- Allow selecting different domains (production, sandbox, or custom) when running `SFDX: Authorize an Org` ([PR #818](https://github.com/forcedotcom/salesforcedx-vscode/pull/818), [Issue #610](https://github.com/forcedotcom/salesforcedx-vscode/issues/610))
+
+- Deploy code on save ([PR #822](https://github.com/forcedotcom/salesforcedx-vscode/pull/822), [Issue #577](https://github.com/forcedotcom/salesforcedx-vscode/issues/577), [Issue #662](https://github.com/forcedotcom/salesforcedx-vscode/issues/662))
+
+### Fixed
+
+#### salesforcedx-vscode-core
+
+- Enable multi-line SOQL selection for `SFDX: Execute SOQL Query with Currently Selected Text` ([PR #833](https://github.com/forcedotcom/salesforcedx-vscode/pull/833/files), [Issue #816](https://github.com/forcedotcom/salesforcedx-vscode/issues/816)). Contribution by [@boxfoot](https://github.com/boxfoot).
+
+- Fix syntax hightlighting for manifest XML files ([PR #823](https://github.com/forcedotcom/salesforcedx-vscode/pull/823))
+
+## 44.11.0 - December
```

Comment:
Shouldn't this include a specific date?

Reply:
Yes, good catch, it should be December 13, 2018

Reply:
Cool, I'll fix this when I make proofreading edits.

Reply:
Thanks